.github/workflows/pr-validate.yml

```
@@ -0,0 +1,16 @@
+name: "Validate PR"
+
+on:
+  pull_request:
```

Based on [this description](https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#event-triggers), I would think it better to use `pull_request_target` here so that it'll work when PRs are received from forks. Thoughts?

```suggestion
on:
  pull_request_target:
    types:
      - opened
      - edited
      - synchronize
      - reopened
```

---

I tried that first but for some reason it didn't seem to work. Maybe it needs both, I'll read up in more details on what the target type does

---

I think it doesn't work for the branch you're experimenting in, since it takes the workflow from the target branch? I think it should work post-merge though